src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
I don't like this. What if user uses a channel name that isn't compatible with channel output data? User could subvert node type use drastically, leading to a complete mess - using a gyro node to get acceleration info, for instance.
Soletta approach on node type is to provide node for each information that can be extracted from device. I guess that this is motivated by #2244. But I believe that we need to understand what information is being exposed there. Maybe it's a case for new node types.
The indices approach used before was really useful: same kind of information being exposed via different channels, because they were independent.